runtime/lib/ttnn/program.cpp

```diff
@@ -295,6 +295,17 @@ run(::tt::target::ttnn::ReductionOp const *op, ::ttnn::Device &device,
   }
 }
 
+static void
+run(::tt::target::ttnn::EmbeddingOp const *op, ::ttnn::device::Device &device,
```

Comment:
Can we use ::ttnn::Device here?

Reply:
No problem, I copied off the other ops here, I'll change them as well.

Reply:
Thanks, just trying to keep this consistent